utils/axios-helper.ts

```diff
@@ -18,4 +18,4 @@ authAxios.interceptors.response.use(
   }
 )
 
-export { authAxios }
\ No newline at end of file
+export { authAxios }
```

Comment:
what changed?

Reply:
I added a CR at the end of the file. The linter config in this project may not catch this, but it does in our other UI projects.

This is minor tech debt.